pkg/controller/operators/labeller/filters.go

```diff
@@ -177,7 +178,12 @@ func Validate(ctx context.Context, logger *logrus.Logger, metadataClient metadat
 				logger.WithFields(logrus.Fields{
 					"gvr":           gvr.String(),
 					"nonconforming": count,
-				}).Info("found nonconforming items")
+				}).Errorf(
+					"found nonconforming items: resource(s) are missing the %q label (metadata.labels[\"%s\"]=\"true\"). "+
+						"You may want to consider deleting it (if this CRD is for an uninstalled solution) or labeling it appropriately.",
+					install.OLMManagedLabelKey,
+					install.OLMManagedLabelKey,
+				)
```

Comment:
Hmm, the phrasing seems awkward to me, how about?:
```go
}).Errorf(
	"Found resources missing the required label %q (`metadata.labels[%s] = \"true\"`). Please label them appropriately, or consider deleting them if they relate to a CRD from an uninstalled solution.",
	install.OLMManagedLabelKey,
	install.OLMManagedLabelKey,
)
```

Reply:
I agree, and I don't think we need the remediation in the log itself.

Reply:
@tmshort so just?:
```go
}).Errorf(
	"Found resources missing the required label %q (`metadata.labels[%s] = \"true\"`),
	install.OLMManagedLabelKey,
	install.OLMManagedLabelKey,
)
```

Reply:
@bentito, I think your version is better (but still has a type), but it's probably not needed?
Is there a document somewhere with this information?
If this occurs, I think it's safer not to offer remediation, because the advice might change over time (whereas the code won't).

Reply:
If we are going to include a suggestion, propose it as an action rather than familiarly.
```
"Resources (%s) missing required label %q (`metadata.labels[%s] = \"true\"`). Add the label or delete unused resources.",
        <list of missing labels>
	install.OLMManagedLabelKey,
	install.OLMManagedLabelKey,
```
Or something similar. It would be good to list the resources.

Reply:
It is a log so it already has the info about the resource see:
```
logger.WithFields(logrus.Fields{
					"gvr":           gvr.String(), <- HERE IS THE INFO
					"nonconforming": count,
				}).Info("found nonconforming items")
```
Therefore, the suggestion above does not seems fit
By last we should NOT remove the text `"found nonconforming items` that is how people find the KCS.